plugins/catalog-backend/src/modules/core/PlaceholderProcessor.ts

```diff
@@ -246,10 +247,15 @@ function relativeUrl({
   try {
     return resolveUrl(value, baseUrl);
   } catch (e) {
-    // The only remaining case that isn't support is a relative file path that should be
-    // resolved using a relative file location. Accessing local file paths can lead to
-    // path traversal attacks and access to any file on the host system. Implementing this
-    // would require additional security measures.
+    if (!path.isAbsolute(value)) {
```

Comment:
Is this a case for using [this](https://github.com/backstage/backstage/blob/d6599a72239909f376061606da941a3985d6ae66/packages/backend-common/src/paths.ts#L50)? I don't think your code below is sufficient as it doesn't collapse / resolve path traversals like `../../../etc/passwd`, but I could be wrong.

Reply:
It does work for such path, I can make a test for that too if necessary